packages/platforms/browser/lib/span-attributes-source.ts

```diff
@@ -4,7 +4,6 @@ const createSpanAttributesSource = (title: string, url: string): SpanAttributesS
   return () => {
     const spanAttributes = new Map<string, SpanAttribute>()
     spanAttributes.set('bugsnag.span.category', 'custom')
-    spanAttributes.set('bugsnag.span.first_class', true)
```

Comment:
Is this right? I thought that unless it's set in SpanOptions it should be omitted?

Reply:
Yeah, this is deleting the line that added it to every span

Reply:
This is what happens when you start doing code reviews before coffee

Reply:
I thought that might be the case lol